addons/web/static/src/js/views/sample_server.js

```diff
@@ -360,7 +361,7 @@ odoo.define('web.SampleServer', function (require) {
                     normalizedGroupBys.push(gb);
                 }
             }
-            const groups = _.groupBy(records, (record) => {
+            const groups = utils.groupBy(records, (record) => {
```

Comment:
instead of returning vals.joint(','), we could maybe use a JSON.strinfify of it
if texts values are used, bad things can happen I think

Reply:
and for this?

Reply:
I think it wouldn't matter since we only want to compare and group records with strictly equal values. If these values hold comas or pound signs for example they would still return the same group name